src/ducks/transactions/TransactionPageErrors/errors.js

```diff
@@ -19,16 +20,22 @@ export const getTransactionPageErrors = ({ triggerCol, accounts }) => {
   )
   const triggers = triggerCol.data
 
+  const bankKonnectorTriggers = triggers
+    .filter(triggerUtils.isKonnectorWorker)
+    .filter(isBankTrigger)
   const erroredTriggers = sortBy(
-    triggers
-      .filter(triggerUtils.isKonnectorWorker)
-      .filter(isBankTrigger)
+    bankKonnectorTriggers
       .filter(triggerStates.isErrored)
       .filter(triggerUtils.hasActionableError)
       .filter(tr => konnectorToAccounts[tr.message.konnector]),
     tr => tr.message.konnector
   )
 
+  if (!erroredTriggers.length > 0 && flag('banks.debug.force-trigger-error')) {
+    erroredTriggers.push(bankKonnectorTriggers[0])
+    erroredTriggers.push(bankKonnectorTriggers[0])
+  }
+
```

Comment:
Si ce n'est pas déjà le cas, ce serait peut-être intéressant de rajouter un test pour `bankKonnectorTriggers` `erroredTriggers` ?

Reply:
ok bon point.

Reply:
Turns out it's already done in TransactionPageErrors.spec.jsx :)